python/ql/lib/semmle/python/dataflow/new/internal/DataFlowDispatchPointsTo.qll

```diff
@@ -421,22 +421,24 @@ class LibraryCallableValue extends DataFlowCallable, TLibraryCallable {
  * TODO: Add `TClassMethodCall` mapping `cls` appropriately.
  */
 newtype TDataFlowCall =
-  TFunctionCall(CallNode call) { call = any(FunctionValue f).getAFunctionCall() } or
-  /** Bound methods need to make room for the explicit self parameter */
-  TMethodCall(CallNode call) { call = any(FunctionValue f).getAMethodCall() } or
-  TClassCall(CallNode call) { call = any(ClassValue c | not c.isAbsent()).getACall() } or
+  /**
+   * Includes function calls, method calls, class calls and library calls.
+   * All these will be associated with a `CallNode`.
+   */
+  TNonSpecialCall(CallNode call) or
```

Comment:
Something about `NonSpecial...` rubs me the wrong way. Could we use a positive formulation instead? I'm tempted to suggest `CallNodeCall`, though that is a bit of a mouthful. (However, it does precisely say that these are calls that happen at call nodes.)

Alternatively, since we have `SpecialCall`, perhaps its counterpart should be `NormalCall`?